Source/Core/PolygonGeometry.js

```diff
@@ -54,6 +54,10 @@ define([
     var computeBoundingRectangleQuaternion = new Quaternion();
     var computeBoundingRectangleMatrix3 = new Matrix3();
 
+    function zeroToOne(n){
```

Comment:
Why not use clamp? https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Math.js#L684